configure

```diff
@@ -852,6 +852,7 @@ probe_need CFG_CMAKE cmake
 # probe for it only in this case.
 if [ -n "$CFG_ANTLR4" ]
 then
+   putvar "CFG_ANTLR4_JAR                     $(echo -n $(find /usr/ -name antlr-complete.jar 2>/dev/null | head -n 1))"
```

Comment:
I have a strange bug: With this line i save the `CFG_ANTLR4_JAR` to the generated config but it also generates an additional line after this entry. 
That entry is empty and stops the use of the `config.mk`.

Do you have an idea?